### ISSUE TITLE
AG WG Review: 2.5.7 Dragging Movements WCAG2ICT guidance

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#268]
## Summary
Review of proposed WCAG2ICT guidance for 2.5.7 Dragging Movements, as incorporated into the Editor's draft. This is new content.

## Maturity Level
Refining

## Action Needed
* Review content
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  All SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
----

## Content for Review

* Section [Applying 2.5.7 Dragging Movements to Non-Web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-2-5-7-dragging-movements-to-non-web-documents-and-software).

## Reference Material
In case you need them, links to various resources on 2.5.7

* [2.5.7 Dragging Movements](https://www.w3.org/TR/WCAG22/#dragging-movements) SC text in WCAG 2.2
* [Intent section of Understanding 2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html#intent)

**Comment by pday1** [comment]
Link to the Applying section didn't work for me, so I navigated to SC 2.5.7, then scrolled down to the Applying section below.
Sharing in case this helps somebody else...
[SC 2.5.7 Dragging Movements](https://w3c.github.io/wcag2ict/#dragging-movements)

**Comment by mbgower** [comment]
I'm a little surprised there isn't more wording around the applicability to platform software.

> This requirement applies to [[software](https://w3c.github.io/wcag2ict/#software)] that interprets pointer actions (i.e. this does not apply to actions that are required to operate the [underlying platform software] or assistive technology).

For non-web software considerations, why _wouldn't_ it apply to platform software?

**Comment by mraccess77** [comment]
I agree it should apply to platform software - I think what we want to exclude is software which uses platform features which the software doesn't have control over.

**Comment by mbgower** [comment]
> I agree it should apply to platform software - I think what we want to exclude is software which uses platform features which the software doesn't have control over.

So I think an additional note (or line) covering platform software?

**Comment by patrickhlauke** [comment]
Agree with @mbgower and @mraccess77 here

**Comment by maryjom** [comment]
See issue #284 where adjustments to the notes for all SC that have a similar note is being proposed for the TF to agree on. This way it'll be consistently applied to all SC that have interpretation of pointer actions.

**Comment by maryjom** [comment]
FYI, the WCAG2ICT Task Force approved the changes to the notes. See PR #296.

**Comment by maryjom** [comment]
I'm going to close this issue. Concerns raised are handled by content changes have been incorporated into the WCAG2ICT editor's draft per issue 296 which has been reviewed by Mike and the task force. If there are any further comments, please open a new issue.
